core/server/api/v2/utils/serializers/input/options.js

```diff
@@ -0,0 +1,43 @@
+const _ = require('lodash');
+
+const trimAndLowerCase = (params) => {
+    params = params || '';
+    if (_.isString(params)) {
+        params = params.split(',');
+    }
+
+    return params.map((item) => {
+        return item.trim().toLowerCase();
+    });
+};
+
+module.exports = {
+    all(apiConfig, options, local = {forModel: true}) {
+        options.modelOptions = _.cloneDeep(options.apiOptions);
+
+        if (options.modelOptions.include) {
+            // No need to validate the values again, happened already in validators
+            options.modelOptions.include = trimAndLowerCase(options.modelOptions.include);
+
+            if (local.forModel) {
+                options.modelOptions.withRelated = options.modelOptions.include;
+                delete options.modelOptions.include;
+            }
+        }
+
+        if (options.modelOptions.fields) {
```

Comment:
Maybe just doing:
```javascript
if (options.modelOptions.fields) {
    options.modelOptions.columns = trimAndLowerCase(options.modelOptions.fields);
    delete options.modelOptions.fields;
}
```
would be a bit more simpler/readable :thinking: ?

Reply:
Yes agree 👍

Reply:
Have updated this in my local commits, commits are coming in a bit.